archives/2019.md

```diff
@@ -7,3 +7,4 @@
 | Conference | Location | Date | Submission Deadline | Flight / Hotel? | CoC? |
 | ---------- | -------- | ---- | ------------------- | ------------------------ | ---- |
 | [CodeMash](http://www.codemash.org/) | Sandusky, Ohio | January 8 - 11, 2019 | August 31, 2018 | Hotel | [Yes](http://www.codemash.org/codemash-code-conduct/) |
+| [JS Kongress Munich - (r)evolution of Architectures](https://js-kongress.com/) | Munich, Germany | March 11 - 12, 2019 | October 14, 2018 | Flight + Hotel | [Yes](https://2017.js-kongress.de/code-of-conduct/) |
```

Comment:
Can you please create a new table for March and put it there? Right now it's w/in the January table.

Also do you mind just naming it just `JS Kongress` and leaving off the rest of it?